app/views/accounts/_course_details.html.erb

```diff
@@ -89,10 +89,10 @@
           </dt>
           <dd class="govuk-summary-list__value">
               <% if application.participant_outcome_state.capitalize == "Passed" %>
-                <strong class="govuk-tag govuk-tag--green">PASSED</strong>
+                <strong class="govuk-tag govuk-tag--green">Passed</strong>
```

Comment:
We can use govuk-components here, it's already installed and set up:

```suggestion
                <%= govuk_tag(text: "Passed", colour: "green") %>
```

Reply:
OK thanks I'll update them

Reply:
Cheers, [here's a link to the gem docs](https://govuk-components.netlify.app/components/tag/).